public/v4/apps/cronicle.yml

```diff
@@ -0,0 +1,41 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: soulteary/cronicle:$$cap_version
+        restart: always
+        volumes:
+            - /etc/localtime:/etc/localtime:ro
+            - /etc/timezone:/etc/timezone:ro
+            - $$cap_appname:/opt/cronicle/data:rw
+            - $$cap_appname-plugins:/opt/cronicle/plugins:rw
```

Comment:
Thanks!

CapRover doesn't read the permission modes `:ro` etc... I'll update after merge.

Reply:
Oh, thanks! Didn't know that! 😃